app/controllers/manageBusinesses/add/AddIncomeSourceStartDateController.scala

```diff
@@ -204,8 +204,8 @@ class AddIncomeSourceStartDateController @Inject()(val authorisedFunctions: Auth
                          isChange: Boolean): String = {
 
     ((isAgent, isChange, incomeSourceType) match {
-      case (false, false, SelfEmployment) => routes.AddBusinessNameController.show()
-      case (_, false, SelfEmployment) => routes.AddBusinessNameController.showAgent()
+      case (false, false, SelfEmployment) => routes.AddBusinessNameController.show(isAgent = false, isChange = false)
```

Comment:
can this down to line 212 be shortened perhaps:

`
((isAgent, isChange, incomeSourceType) match {
case (true, true, _) => routes.IncomeSourceCheckDetailsController.showAgent(incomeSourceType)
case (false, true, _) => routes.IncomeSourceCheckDetailsController.show(incomeSourceType)
case (_, _, SelfEmployment) => routes.AddBusinessNameController.show(isAgent = isAgent, isChange = isChange)
case (_, _, _) => controllers.manageBusinesses.add.routes.AddPropertyController.show(isAgent = isAgent)
`

Reply:
Why the last case is AddPropertyController? This was not used before my changes

Reply:
I think this AddPropertyController was already in the code for this scenario.

My meaning behind this thread is that we can remove a few lines of code by not caring so much about the contents of the isChange and isAgent values, since we can just put them straight in like this 
`.show(isAgent = isAgent, isChange = isChange)`

Reply:
You are right my bad, I was looking at incomeSources/AddIncomeSourceStartDateController. Got your point for reducing it and passing the value of the parameter, I changed it and is working👍🏻